tasks/task.go

```diff
@@ -0,0 +1,13 @@
+package tasks
+
+import (
+	"github.com/beego/beego/v2/task"
+)
+
+func init() {
```

Comment:
I am a bit concerned to just `init` here as it could be a bit hidden for future developers (it's not in `main.go`) 🤔 How about documenting it in the README as well that the background tasks are initialized in `/tasks`?

Reply:
I've updated the `README.md` to have more detail about `background task` already ✌️